test/IRGen/unexploded-calls.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %swift -target thumbv7-unknown-windows-msvc -parse-stdlib -parse-as-library -I %S/Inputs/usr/include -module-name Swift -S -emit-ir -o - %s | %FileCheck %s
```

Comment:
@compnerd This test is failing in PR testing

Reply:
Interesting.  Wonder how it passed during this run.  Do we not have the ARM backend enabled?  The warning originates from clang (its not a swift side thing).  I'll look into this.